src/main.cpp

```diff
@@ -358,6 +358,13 @@ bool CInputStreamAdaptive::OpenStream(int streamid)
       }
     }
 
+    if (track->GetSampleDescription(0) == NULL)
```

Comment:
compare NULL is not needed
```suggestion
    if (!track->GetSampleDescription(0))
```